test/e2e/scheduling/predicates.go

```diff
@@ -554,7 +554,14 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		verifyResult(cs, 1, 0, ns)
 	})
 
-	ginkgo.It("validates that there is no conflict between pods with same hostPort but different hostIP and protocol", func() {
+	/*
+		Release : v1.15
+		Testname: Services, HostPort matching and HostIP and Protocol not-matching
```

Comment:
This test only tests the scheduler and how it handles host ports. It has nothing to do with services. The description is also incorrect. These pods are scheduled to the same node. A better name and description would be:
```
Testname: Scheduling, HostPort matching but HostIP or Protocol is different
Description: Pods with the same HostPort value MUST be able to be scheduled to the same node if the HostIP or Protocol is different.
```